nrf52-rfm69/ChirpFMRxTx.h

```diff
@@ -2,6 +2,6 @@ class ChirpFMRxTx
 {
```

Comment:
@s-axford @DelightedWaif @johnnyb8 this is def a preference but thoughts on having the interface at this level? maybe up one? or only move it when we have more hardware implementations?